src/index.js

```diff
@@ -1,14 +1,43 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 import '@fortawesome/fontawesome-free/js/all.js';
 
-function component() {
-  const element = document.createElement('div');
+// function component() {
+//   const element = document.createElement('div');
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+//   // Lodash, now imported by this script
+//   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
```

Comment:
- [ ] As mentioned above, do not commit old pieces of code as an inline comments.